.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I believe this was mistakenly included! Could you remove this change from your commit?